### PR TITLE
Fix toggle if state = unavailable

### PR DESCRIPTION
**Pull request by c727**
fix: https://github.com/home-assistant/ui-schema/issues/128

### CODE REVIEW

File: src/panels/lovelace/entity-rows/hui-toggle-entity-row.js [hunk 1]
```diff
@@ -4,29 +4,62 @@ import { PolymerElement } from '@polymer/polymer/polymer-element.js';
...
+      },
+      _canToggle: {
+        type: Boolean,
+        computed: '_computeCanToggle(_stateObj.state)'
+      }
     };
   }
 
   _computeStateObj(states, entityId) {
     return states && entityId in states ? states[entityId] : null;
```
Comment (c727) [inline src/panels/lovelace/entity-rows/hui-toggle-entity-row.js]: this is not guarded by the stateObj-check in hui-generic-entity-row. other cards will also need updates

File: src/panels/lovelace/entity-rows/hui-toggle-entity-row.js [hunk 2]
```diff
@@ -4,29 +4,62 @@ import { PolymerElement } from '@polymer/polymer/polymer-element.js';
...
   static get properties() {
     return {
       hass: Object,
-      _config: Object
+      _config: Object,
+      _stateObj: {
+        type: Object,
+        computed: '_computeStateObj(hass.states, _config.entity)'
+      },
+      _canToggle: {
```
Comment (balloob) [inline src/panels/lovelace/entity-rows/hui-toggle-entity-row.js]: This feels a bit silly for a toggle card. What's the use case of rendering a non toggle able entity in a toggle card? Shouldn't this just be an error?

Comment (c727) [reply src/panels/lovelace/entity-rows/hui-toggle-entity-row.js]: Have you noticed the linked issue?

Comment (balloob) [reply src/panels/lovelace/entity-rows/hui-toggle-entity-row.js]: Oh, unavailable.

I was on my phone 👼

**Pull request merged**